### DISCUSSION TITLE
Can i host dashboard on heroku? my bot is hosting on heroku

### DISCUSSION THREAD

**Discussion by DIABL06666** [Androz2091/AtlantaBot#633] [Q&A] [answered]
I want host dashboard on heroku but idk how

**Comment by DIABL06666** [comment, marked as answer]
Nevermind, i solved this

**Reply by AEmeoow** [reply]
Could you please help me to do this?

**Reply by AEmeoow** [reply]
And just hosting in general, as the self-host guide won't work for me, it just won't load the page

**Reply by ombhatt2** [reply]
How did you do it?
